jni_bindings/src/lib.rs

```diff
@@ -457,273 +536,268 @@ pub extern "system" fn Java_org_phoenixframework_liveview_lib_Document_get_1node
             env.set_object_array_element(&array, i as jsize, &java_object)
                 .unwrap();
         }
+
         array
-    });
-
-    match result {
-        Ok(jobject_array) => jobject_array,
-        Err(err) => {
-            let mut env = env_mutex.lock().unwrap();
-            let exception_class = env.find_class("java/lang/RuntimeException").unwrap();
-            let message = format!(
-                "Rust panic occurred in Document get_1node_1element_1attributes: {:?}",
-                err
-            );
-            env.throw_new(exception_class, message).unwrap();
-            JObject::null().into()
-        }
     }
 }
 
 #[no_mangle]
-pub extern "system" fn Java_org_phoenixframework_liveview_lib_Attribute_drop(
+pub unsafe extern "system" fn Java_org_phoenixframework_liveview_lib_Attribute_drop(
     mut env: JNIEnv,
     _: JClass,
+    // non-null raw pointer to Attribute
```

Comment:
The `non-null` can't be true or there would be no need for the `this.is_null()` check below.
```suggestion
    // raw pointer to Attribute
```